### PR TITLE
Edited index.md in architecture_overview folder

### PR DESCRIPTION
**Pull request by philip-menzies**

### CODE REVIEW

File: docs/get_started/architecture_overview/index.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-# Architecture Overview
+# Architecture overview
 
-HCL Digital Experience is a solid and reliable platform that can help deliver the vital services of your organization. It is designed to be scalable and flexible, supports authentication for security and personalization, and aids in easy integration with varied applications. Strong encryption and cross-system authentication keep your business-critical functions safe. Your teams can easily create, manage, and deliver powerful and reliable digital experiences everyday.
+HCL Digital Experience is a platform that can help you deliver the critical services of your organization. It is designed to be scalable and flexible, supports authentication for security and personalization, and aids in  integration with varied applications. Strong encryption and cross-system authentication keep your business-critical functions safe. Your teams can create, manage, and deliver powerful and reliable digital experiences every day.
```
Comment (biancaluna) [inline docs/get_started/architecture_overview/index.md:3]:
```suggestion
HCL Digital Experience is a platform that helps you deliver the critical services of your organization. It is designed to be scalable and flexible, supports authentication for security and personalization, and aids in  integration with varied applications. Strong encryption and cross-system authentication keep your business-critical functions safe. Your teams can create, manage, and deliver powerful and reliable digital experiences every day.
```

File: docs/get_started/architecture_overview/index.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+- **Traditional deployments**: You can deploy HCL DX on-premises in the WebSphere Application Server infrastructure.
+- **Container-based deployments**: HCL supports container-based deployments. You can deploy HCL DX into Kubernetes infrastructures to manage and maintain multiple environments such as testing, development, staging, and production.
 
 For more information, refer to the detailed system requirements for [traditional deployments](../system_requirements/traditional/index.md) or [container-based deployments](../system_requirements/kubernetes/kubernetes-runtime.md).
 
 ![Containerization Architecture Overview](../../images/HCL-DX-deployment-diagram-Kubernetes.png)
 
-There is one component called DX Core that is common for both deployment options. DX Core contains functionalities, which were originally available for traditional deployments, that are now applicable for both traditional and container-based deployments.
+One component, DX Core, is common to both deployment options. DX Core contains several functions, which were originally available for traditional deployments. Those functions applicable to both traditional and container-based deployments.
```
Comment (biancaluna) [inline docs/get_started/architecture_overview/index.md:14]:
```suggestion
One component, DX Core, is common to both deployment options. DX Core contains several functions, which were originally available for traditional deployments. These functions are applicable to both traditional and container-based deployments.
```

**Review by biancaluna — COMMENTED** [review]
Hi @philip-menzies , please see suggested changes. Thanks!

File: docs/get_started/architecture_overview/index.md (hunk 1 above)
Comment (philip-menzies) [reply docs/get_started/architecture_overview/index.md:3]: OK

File: docs/get_started/architecture_overview/index.md (hunk 2 above)
Comment (philip-menzies) [reply docs/get_started/architecture_overview/index.md:14]: OK

**Review by biancaluna — APPROVED** [review]

**Pull request merged**